project-base/storefront/pages/search.tsx

```diff
@@ -48,12 +51,14 @@ export const getServerSideProps = getServerSidePropsWrapper(({ redisClient, doma
         return redirect;
     }
 
-    return initServerSideProps({
-        context,
-        redisClient,
-        domainConfig,
-        t,
-    });
+    return initServerSideProps<TypeSearchQueryVariables | TypeSearchProductsQueryVariables | TypeAdvertsQueryVariables>(
```

Comment:
These types are used when you add prefetched queries. They help you catch mistakes in used variables, etc. However, you do not use any prefetched queries here, so I would say they are unnecessary